### ISSUE TITLE
Improve SQLClient documentation for create_table and insert_table methods

### ISSUE THREAD

**Issue by oriolbx** [CartoDB/cartoframes#1417]
The methods create_table and insert_table the SQLClient documentation are defined as:

<img width="742" alt="Screenshot 2019-12-27 at 13 59 31" src="https://user-images.githubusercontent.com/11177693/71518136-aae9e880-28b1-11ea-9dbf-f87138b266db.png">

<img width="387" alt="Screenshot 2019-12-27 at 14 00 27" src="https://user-images.githubusercontent.com/11177693/71518141-af160600-28b1-11ea-9591-598ed90ec018.png">

Both methods don't indicate if the input argument `columns` should be an array, string or any other data type. On the other hand, the `insert_table()` method doesn't have any kind of explanation of what it does.

The lack of information is causing a bad use of these two methods and a bad result. For example, if I use the insert_table method in the next way:

```python
sql.insert_table('stores_df', 'name', 'miau')
```
The error returned is the next one:

```
Batch SQL job failed with result: {'user': 'oboix', 'status': 'failed', 'query': "INSERT INTO stores_df 
(n,a,m,e) VALUES('m','i','a','u');", 'created_at': '2019-12-27T13:02:59.115Z', 'updated_at': '2019-12-
27T13:02:59.131Z', 'failed_reason': 'column "n" of relation "stores_df" does not exist', 'job_id': 
'0cc54b06-6fc1-4525-aeb6-34ac4abd7e27'}
```

**Comment by Jesus89** [comment]
I will tackle this in https://github.com/CartoDB/cartoframes/pull/1409

**Comment by Jesus89** [comment]
Solved in #1409.
